packages/cli/src/presentation.ts

```diff
@@ -196,3 +196,56 @@ presentation
       console.error('Presentation could not be verified.')
     }
   })
+
+presentation
+  .command('output')
+  .description('Print W3C Verifiable Presentation to stdout')
+  .option('-t, --tag <string>', 'Optional. Specify the tag for the presentation.')
+  .action(async (options) => {
+    const agent = getAgent(program.opts().config)
+
+    const presentations = await agent.dataStoreORMGetVerifiablePresentations({})
+
+    if (presentations.length > 0) {
+
+      let selected = null
+      const list: any = []
+      if (options.tag) {
+        const matches = presentations.filter(pres => pres.verifiablePresentation.tag === options.tag)
+        if (matches.length > 1) {
+          console.log('Found multiple matching presentations. Only showing the first one.')
+        }
+        selected = matches[0]
+      } else {
+        for (const pres of presentations) {
+          list.push({
+            name:
+              'Issuance Date: ' +
+              pres.verifiablePresentation.issuanceDate +
+              ' | Holder: ' +
+              pres.verifiablePresentation.holder +
+              ' | Tag: ' +
+              pres.verifiablePresentation.tag,
+            value: pres,
+          })
+        }
+        const answers = await inquirer.prompt([
+          {
+            type: 'list',
+            name: 'presentation',
+            choices: list,
+            message: 'Select presentation',
+          },
+        ])
+        selected = answers.presentation
+      }
+
+      if (selected) {
+        console.dir(selected, { depth: 10 })
```

Comment:
Maybe we should use `console.log(JSON.stringify(selected, null, 2))` instead, so that the putput is a valid json?

Reply:
Good point

Reply:
OK, I'll make that change.

Reply:
I have made that change.

Just note that:
- The `console.dir` command already prints in nicely-formatted JSON with indentation. As a matter-of-fact: it prints in my terminal in such a way that the output is colored, so I think that version is even more fancy and potentially nicer for the user, so it's worth a look to see if you want to change it back.
- The `console.dir` approach is already used in many places in the codebase.